web/src/stream_list.ts

```diff
@@ -839,6 +841,31 @@ export function set_event_handlers({
 }: {
     on_stream_click: (stream_id: number, trigger: string) => void;
 }): void {
+    $("#stream_filters").on(
+        "mouseenter",
+        "li .subscription_block .stream-name",
+        function (this: HTMLElement, e) {
+            const stream_id = stream_id_for_elt($(e.target).parents("li"));
```

Comment:
This `parents` check would benefit from including a class to avoid this code being fragile with any future refactoring. `li.narrow-filter` seems fine.

Reply:
Ah yeah I was mostly just exactly copying this from the `$("#stream_filters").on("click"` below, but I can update both of them.